controller/lender_providers/index.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
why do we need lender_providers controller and model and a separate table, when we already have LoanProvider controller, model and table. Whats the difference?

Reply:
in loan provider table row name is different , in this name is max_tenure , min_amount but according to previous task I got table name data is provider name,loan_amount,tenure,roi,eligibilty